openapi.yaml

```diff
@@ -5068,13 +5068,17 @@ components:
         jira:
           $ref: '#/components/schemas/JIRAFilter'
         updated_from:
-          description: 'Paginate the response: the returned PRs are updated no older
-            than this timestamp, including the beginning.'
+          description: 'Paginate the response: the returned PRs are updated on GitHub no older
+            than this timestamp, including the beginning. Note: "updated on GitHub" means that
+            we don´t consider release or deployment timestamps. Use this field only to
+            paginate after calling `/paginate/pull_requests`. Subject of future changes.'
```

Comment:
Ok I see this is to clarify [DEV-4875](https://athenianco.atlassian.net/browse/DEV-4875)

Reply:
Yes. In fact, I made a fully-featured 10-page presentation for today's demo about it. They trolled me well.